lib/cpuinfo.c

```diff
@@ -343,7 +347,7 @@ detect_apic_masks(struct apic_info *apic)
         if (detect_apic_core_masks(apic) != 0)
                 return -2;
 
-        if (detect_apic_cache_masks(apic) != 0)
+	if (detect_apic_cache_masks(apic, 4) != 0)
```

Comment:
Magic number. Cen We add some define for it? For example DETERMINISTIC_CACHE_PARAMETERS_EAX